src/frontend/src/flows/register.ts

```diff
@@ -16,22 +16,36 @@ import { nextTick } from "process";
 import { icLogo } from "../components/icons";
 
 const pageContent = html`
-  <div class="container">
-    <h1>Create a new Internet Identity Anchor</h1>
-    <form id="registerForm">
-      <p>Please provide a name for your device.</p>
-      <input id="registerAlias" placeholder="Device name" />
-      <button type="submit" class="primary">Create</button>
-      <button id="registerCancel" type="button">Cancel</button>
+  <div class="l-container c-card c-card--highlight">
+    <hgroup>
+      <h1 class="t-title t-title--main">
+        Create a new Internet Identity Anchor
+      </h1>
+      <p class="t-lead">Please provide a name for your device.</p>
+    </hgroup>
+    <form id="registerForm" class="l-section">
+      <input id="registerAlias" placeholder="Device name" class="c-input" />
+      <div class="l-section">
+        <button type="submit" class="c-button">Create</button>
+        <button
+          id="registerCancel"
+          type="button"
+          class="c-button c-button--secondary"
+        >
+          Cancel
+        </button>
+      </div>
     </form>
   </div>
 `;
 
 const constructingContent = html`
-  <div class="container flex center">
-    <h1>Constructing new Identity Anchor</h1>
+  <div class="l-container c-card  c-card--highlight">
```

Comment:
there are two spaces between the class. prettier did not detect it?

Reply:
I fixed it. But I think prettier has some issues with HTML strings in JS... (It obviously does not catch everything. Should we create an issue for that?)

Reply:
Thanks. yes would be nice, not sure there is a solution but it's cool if formatting works out for anything